app/views/user_mailer/password_reset.html.erb

```diff
@@ -1,5 +1,9 @@
-<h1>User#password_reset</h1>
+<h1><%= t "user_mailer.password_reset.h1" %></h1>
 
-<p>
-  <%= @greeting %>, find me in app/views/user_mailer/password_reset.html.erb
-</p>
+<p><%= t "user_mailer.password_reset.content_p1" %></p>
+
+<%= link_to t("user_mailer.password_reset.title_url"), edit_password_reset_url(@user.reset_token, email: @user.email) %>
```

Comment:
dùng edit_password_reset_path thay edit_password_reset_url được không em?

Reply:
Dạ không ạ,  _path chỉ sử dụng trong trường hợp local thôi mình dùng thì được nhưng gửi link cho người khác thì sẽ không tìm thấy ,_url  nó chứa domain nên hoạt động hoàn toàn trên domain đó